src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -13,6 +13,9 @@
 
 use Psr\Container\ContainerInterface;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Debug\Exception\FatalThrowableError;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
```

Comment:
Why `ContainerBuilder`, and not `Container`?

Reply:
ill use Container